lib/glimesh/streams/channel.ex

```diff
@@ -15,6 +15,8 @@ defmodule Glimesh.Streams.Channel do
     field :stream_key, :string
     field :inaccessible, :boolean, default: false
     field :backend, :string
+    field :disable_hyperlinks, :boolean, default: false
+    field :block_links, :boolean, default: false
```

Comment:
I feel disabling of links being parsed should be done on user side and not channel as channels should only have the option to allow or disallow hyperlinks being posted.